onnxruntime/core/providers/nuphar/nuphar_allocator.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <tvm/tvm.h>
+#include "core/framework/allocator.h"
+
+// TODO retire this
+
+namespace onnxruntime {
+
+constexpr const char* TVM_STACKVM = "TvmStackVm";
+
+class NupharAllocator : public IDeviceAllocator {
+ public:
+  NupharAllocator(TVMContext tvm_ctx) : tvm_ctx_(tvm_ctx) {}
+  virtual void* Alloc(size_t size) override;
+  virtual void Free(void* p) override;
```

Comment:
Just use 'override' and not 'virtual' in derived classes

Reply:
Removed file after taking Ke Zhang's change.

---
In reply to: [313184500](https://github.com/microsoft/onnxruntime/pull/881#discussion_r313184500) [](ancestors = 313184500)